test/DebugInfo/DebugInfoUnsupported.ll

```diff
@@ -0,0 +1,42 @@
+; This test checks that metadata with unknown DWARF operations is translated
+; to "empty" debug info to SPIR-V and back.
+; TODO: remove this test as soon as more DWARF operations are supported.
+
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv %t.spv -to-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; CHECK-SPIRV: DebugInfoNone
```

Comment:
This check doesn't seem really useful, this instruction may be used by many other instructions. I think check for `DebugValue` and its operands makes more sense here.

Reply:
Updated in 715e6bb